packages/p2p/src/components/floating-rate/floating-rate.jsx

```diff
@@ -1,11 +1,11 @@
+import React from 'react';
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
-import React from 'react';
 import { InputField, Text } from '@deriv/components';
 import { formatMoney, isMobile, mobileOSDetect } from '@deriv/shared';
 import { observer, useStore } from '@deriv/stores';
 import { localize } from 'Components/i18next';
-import { useStores } from 'Stores';
+import useMarketRate from '../../hooks/useMarketRate';
```

Comment:
`non-blocking`: could you perhaps set up a default path for the 'Hooks' folder?

Reply:
Sure @ameerul-deriv will update this :pray: